.github/workflows/benchmarks.yml

```diff
@@ -63,17 +63,23 @@ on:
         required: false
         default: ''
         description: 'custom solc version to use for benchmarks'
+      target-machine:
+        type: string
+        required: false
+        default: ''
+        description: 'Target machine passed via `--target` for era-compiler-tester. Available arguments: `EraVM`, `EVM`, `EVMInterpreter`. Use `default` or `` to skip `--target` parameter.'

```

Comment:
These will be `EraVM`, `EVM`, `EVMEmulator` when we start using it, so let's update immediately.

Reply:
Fixed